src/styles/index.scss

```diff
@@ -2,2 +1,77 @@
+* {
   margin: 0;
+  padding: 0;
+}
+
+@mixin circle($size) {
+  width: $size;
+  height: $size;
+  border-radius: 50%;
+}
+
+@mixin watch-hand($width, $height, $color) {
+  width: $width;
+  height: $height;
+  background-color: $color;
+  transform-origin: bottom;
+}
```

Comment:
It's great that you're using mixins to avoid code duplication. But you should also consider adding a `position: relative;` property to this mixin. Since you're using `transform-origin: bottom;` here, it would be better to make sure that the position property is set to relative. This way, the mixin is more versatile and can be used in a wider variety of situations.